bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/config/dto/Availability.java

```diff
@@ -25,6 +25,8 @@ public class Availability {
     protected String payloadAvailable = "online";
     @SerializedName("payload_not_available")
     protected String payloadNotAvailable = "offline";
+    @SerializedName("value_template")
+    protected String valueTemplate;
```

Comment:
Should be Nullable according to documentation https://www.home-assistant.io/integrations/sensor.mqtt/#value_template

Reply:
ah, I was following payloadAvailable not being Nullable, but it makes sense that they're not, because they have a default value.

Reply:
Could you please add Nullable to the getter too.
DTOs are excluded from static code analysis tool and they don't have NonNullByDefault, but it will be useful to know which fields can be null by API documentation.

Reply:
good catch